### PR TITLE
tcpdump: Add -D and -r options

### PR DESCRIPTION
**Pull request by mingchen**
----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ ] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/tcpdump.md [hunk 1]
```diff
@@ -25,3 +29,7 @@
 - Capture all traffic except traffic over port 22 and save to a dump file:
 
 `tcpdump -w dumpfile.pcap not port 22`
```
Comment (agnivade) [inline pages/common/tcpdump.md]: Make dumpfile.pcap and 22 in tokens (`{{}}`)

File: pages/common/tcpdump.md [hunk 2]
```diff
@@ -25,3 +29,7 @@
...
+
+- Read from an exist dump file:
```
Comment (agnivade) [inline pages/common/tcpdump.md]: => `from a given dump file`

File: pages/common/tcpdump.md [hunk 3]
```diff
@@ -25,3 +29,7 @@
...
+
+`tcpdump -r dumpfile.pcap`
```
Comment (agnivade) [inline pages/common/tcpdump.md]: dumpfile.pcap in tokens (`{{}}`)

**Comment by sbrl** [comment]
Thanks, @mingchen! 😺

**Pull request merged**